server/tilesource/tiff_reader.py

```diff
@@ -268,9 +268,10 @@ def _getJpegTables(self):
         tableSize = ctypes.c_uint32()
         tableBuffer = ctypes.c_voidp()
 
-        libtiff_ctypes.libtiff.TIFFGetField.argtypes = \
-            libtiff_ctypes.libtiff.TIFFGetField.argtypes[:2] + \
-            [ctypes.POINTER(ctypes.c_uint32), ctypes.POINTER(ctypes.c_void_p)]
+        if libtiff_ctypes.libtiff.TIFFGetField.argtypes:
+            libtiff_ctypes.libtiff.TIFFGetField.argtypes = \
+                libtiff_ctypes.libtiff.TIFFGetField.argtypes[:2] + \
+                [ctypes.POINTER(ctypes.c_uint32), ctypes.POINTER(ctypes.c_void_p)]
```

Comment:
I'll add one in the future -- I responded to @jbeezley approval my merging already.